docs/README.md

```diff
@@ -39,6 +39,7 @@
   * [Pausing the Test](articles/documentation/test-api/pausing-the-test.md)
   * [Handling Native Dialogs](articles/documentation/test-api/handling-native-dialogs.md)
   * [Working with &lt;iframes&gt;](articles/documentation/test-api/working-with-iframes.md)
+  * [A-Z](articles/documentation/test-api/a-z.md)
```

Comment:
It looks weird here with this name, maybe we can somehow describe it better in title?